docs/Installing.asciidoc

```diff
@@ -67,6 +67,7 @@ From there, you can trigger a new job or clone an existing one, e.g.:
 
 [source,sh]
 ----
+wget https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-example/main/scenario-definitions.yaml
 openqa-cli schedule --monitor \
     --param-file SCENARIO_DEFINITIONS_YAML=scenario-definitions.yaml \
```

Comment:
According to http://open.qa/docs/#_create_and_monitor_openqa_jobs_from_within_the_ci_runner
one should be able to do

SCENARIO_DEFINITIONS_YAML_FILE
```suggestion
openqa-cli schedule --monitor \
    --param-file SCENARIO_DEFINITIONS_YAML_FILE=https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-example/main/scenario-definitions.yaml \
```

then but something feels off here. Why would we have a parameter SCENARIO_DEFINITIONS_YAML and one SCENARIO_DEFINITIONS_YAML_FILE which can take an URL but for that we normally use the suffix _URL?

I suggest to check on that before continuing here. Just downloading a file locally to pass it unaltered to a remote server which then reads the git repo anyway sounds wrong and is not what we should encourage users to do. Maybe that's even a missing feature in openQA which I expected to be present already.

Reply:
We have `SCENARIO_DEFINITIONS_YAML_FILE` because initially it was only able to take a local file. I added the ability to specify a URL later for the webhook-based GitHub integration. And it only works if `async=1` is used (which is a limitation that makes sense).

Note that `SCENARIO_DEFINITIONS_YAML` takes the YAML contents itself. It only works with a file here due to `--param-file` which is a client-side feature - which means the file needs to be downloaded on the client like in @perlpunk wget example. (When using `SCENARIO_DEFINITIONS_YAML_FILE` the path would be located by the server.)

I'd probably prefer https://github.com/os-autoinst/openQA/pull/5829#discussion_r1705530848. Note that your example doesn't work at all. It tells the *client* to lookup the local file `https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-example/main/scenario-definitions.yaml` and upload its contents via `SCENARIO_DEFINITIONS_YAML_FILE`.